core/src/test/java/org/apache/gravitino/storage/relational/service/TestFilesetMetaService.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.gravitino.integration.test.store.relational.service;
+package org.apache.gravitino.storage.relational.service;
```

Comment:
What is the purpose of changing this package name?

Reply:
Because this class is actually a UT, not IT, it tests the interface of `FilesetMetaService`. So I changed the package name